project.md

```diff
@@ -58,11 +113,17 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+About 60% of the time would be spent engaging with the language aspects, while 40% will be working on the systems aspects of this project. There are many language concerns about how much natural language should be incorporated into the DSL as how many customizations should users be able to define and assumptions that the program should make. As more file formats are being parsed and available in this DSL, more implementation will be needed to fit existing libraries (though helpful in heavy-lifting a lot of the system aspects) into the DSL.
+
 ### Scope
 
 _How big an idea is this? How ambitious is this project?_
 
+I think the DSL for this idea is a well-scoped idea for the class, and the inputs and outputs are relatively easy to parse and generate. Since this need is, at its root, a text-processing problem, many parts of the project can be reduced down to strings to simplify the implementation. My reviewer of this project in HW3 also noted that this was the most feasible and scoped idea (of the 3 ideas I had in that homework) and agreed that it would be good for a CS111 project as well.
+
```

Comment:
I actually think that I was your reviewer for HW3 and I still agree with this sentiment. It seems very feasible to complete over the course of this semester and also leaves the option for you to add a handful of additional features after the basic idea is executed and working